content/english/security-phishing/6-Summary/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Summary"
+description: ""
+draft: false
+weight: 6
+---
+	
+### Conclusion
+In conclusion, the common features showed by a Phishing scams has the following one or all the properties:
```

Comment:
In conclusion, the common features found in Phishing scams have at least one of the following properties: